src/site/styles/layout.less

```diff
@@ -83,11 +83,46 @@ input {
     position: relative;
 }
 
-.action-confirmation {
-    position: absolute;
-    top: 0;
-    left: 0;
-    background: white;
+.action-confirmation { 
```

Comment:
Why nest the rules under `.action-confirmation`? You don't need the extra specificity, it just adds CSS bloat.